flashlight.go

```diff
@@ -437,46 +437,140 @@ func (f *Flashlight) Run(httpProxyAddr, socksProxyAddr string,
 	afterStart func(cl *client.Client),
 	onError func(err error),
 ) {
-	stop := f.StartBackgroundServices()
+	// Until we know our country, default to IR which has all detection rules
+	log.Debug("Defaulting detour country to IR until real country is known")
+	detour.SetCountry("IR")
+
+	stop, err := f.StartBackgroundServices()
+	if err != nil {
+		log.Errorf("Unable to start background services: %v", err)
```

Comment:
```suggestion
		log.Errorf("Unable to start background services: %w", err)
```

Reply:
Since we're just logging the error here, we want it to print nicely. Otherwise, if we were returning it then, yes, we would wrap it.